Mage/src/main/java/mage/abilities/effects/common/combat/CantBeBlockedByTargetSourceEffect.java

```diff
@@ -21,6 +21,11 @@ public class CantBeBlockedByTargetSourceEffect extends RestrictionEffect {
 
     public CantBeBlockedByTargetSourceEffect(Duration duration) {
         super(duration);
+        // TODO: this is not great hint. Better than nothing?
```

Comment:
This is the only one using a TargetPointer() to dynamically decide what can or can't block {this}.
Best solution would maybe to provide a staticMessage as an argument?